apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -461,7 +480,11 @@ export default NextAuth({
 
           if (!userWithNewEmail) {
             await prisma.user.update({ where: { id: existingUser.id }, data: { email: user.email } });
-            return true;
+            if (existingUser.twoFactorEnabled) {
+              return `/auth/login?totp=${await signJwt({ email: user.email })}`;
```

Comment:
This is te redirection needed to let the user input a totp code in an external provider use case.